src/twisted/newsfragments/5851.bugfix

```diff
@@ -0,0 +1 @@
+fix t.i.ssl.Certificate __repr__ for certificates without a common name.
```

Comment:
Maybe have it more like a release announcement and less like a commit messaage :)

```suggestion
twisted.internet.ssl.Certificate.__repr__ can now handle certificates without a common name (CN) in the certificate itself or the signing CA.
```